system/lib/gl/webgl1.c

```diff
@@ -624,4 +624,164 @@ RET_SYNC_GL_FUNCTION_3(EM_FUNC_SIG_VIII, void, glGetQueryObjectuivEXT, GLenum, G
 RET_SYNC_GL_FUNCTION_3(EM_FUNC_SIG_VIII, void, glGetQueryObjecti64vEXT, GLenum, GLenum, GLint64 *);
 RET_SYNC_GL_FUNCTION_3(EM_FUNC_SIG_VIII, void, glGetQueryObjectui64vEXT, GLenum, GLenum, GLuint64 *);
 
+#define RETURN_FN(functionName) if (!strcmp(name, #functionName)) return functionName;
+
+void *emscripten_webgl1_get_proc_address(const char *name)
```

Comment:
This (and the webgl2 function too) overlap with the existing gl.c code which also has the full list of gl commands and a strcmp for each of them, etc. I think we should refactor this PR so there is one shared location for both - I'm fine if that's this new code.

Note that the old code has the feature that it is aware of `LEGACY_GL_EMULATION`, which this seems not to be. Otherwise maybe we can just throw the old code out.